src/components/grid-view/GridViewSlot.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+import * as React from 'react';
+import classNames from 'classnames';
+import './GridViewSlot.scss';
+
+type Props = {
+    item: BoxItem,
+    slotIndex: number,
+    slotRenderer: (slotIndex: number) => ?React.Element<any>,
+    slotWidth: string,
+};
+
+const GridViewSlot = ({ item, slotIndex, slotRenderer, slotWidth }: Props) => {
+    const { id, selected } = item;
+
+    // using item's id as key is important for renrendering.  React Virtualized Table rerenders
+    // on every 1px scroll, so using improper key would lead to image flickering in each
+    // card of the grid view when scrolling.
+    return (
+        <div key={id} className="bdl-GridViewSlot" style={{ maxWidth: slotWidth, flexBasis: slotWidth }}>
```

Comment:
Should the key be moved back to where the list of `GridViewSlot` is created? Since there is no list within this component, I'm not sure of the benefits of adding `key` here

Reply:
It might make sense to have `selected` be a prop as well and if we do that we don't even need to pass in the item if id is used for the key on the GridViewSlot itself